rbac/management/principal/cleaner.py

```
@@ -245,6 +357,231 @@ def process_umb_event(frame, umb_client: Stomp, bootstrap_service: TenantBootstr
+            "dry_run": dry_run,
+        }
+        dlq_producer.send_kafka_message(dlq_topic, dlq_message)
+        logger.info(
+            "process_kafka_message: Sent failed message to DLQ topic %s (partition=%d, offset=%d)",
+            dlq_topic,
+            message.partition,
+            message.offset,
+        )
+        # Return success=True so offset gets committed (message moved to DLQ)
+        return MessageProcessingResult(should_continue=True, success=True)
+    except Exception as dlq_error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)